### PR TITLE
Updated menu component to use mds

### PR DESCRIPTION
**Pull request by bexsoft**
## What does this do?

Updated menu component to use mds one. Also updated tests to match new component

## How does it look?

Screenshot 2023-06-12 at 20 50 20
Screenshot 2023-06-12 at 20 46 30
Screenshot 2023-06-12 at 20 46 14
Screenshot 2023-06-12 at 20 46 04

### CODE REVIEW

**Comment by prakashsvmx** [comment]
@bexsoft , i have tested and looks good. :+1: 

an observation is that the menu group expand/collapse icon could be aligned on the same line when sidebar menu is minimised.

image

**Comment by bexsoft** [comment]
> icon could be aligned on the same line when sidebar menu is minimised.

Will review it and commit a fix in mds repo

**Review by jinapurapu — CHANGES_REQUESTED** [review]

File: portal-ui/src/screens/Console/Menu/MenuWrapper.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
+// This file is part of MinIO Console Server
+// Copyright (c) 2022 MinIO, Inc.
```
[the remark this answers is not in the document]
Comment (bexsoft) [reply portal-ui/src/screens/Console/Menu/MenuWrapper.tsx]: Applied suggestion

**Review by jinapurapu — APPROVED** [review]

**Review by kaankabalak — APPROVED** [review]
LGTM

**Review by cesnietor — APPROVED** [review]
tested LGTM

**Pull request merged**